@here/harp-text-canvas/lib/rendering/FontCatalog.ts

```diff
@@ -502,6 +503,34 @@ export class FontCatalog {
         })!;
     }
 
+    /**
+     * Update the info with the memory footprint caused by objects owned by the `FontCatalog`.
+     *
+     * @param info The info object to increment with the values from this `FontCatalog`.
+     */
+    updateMemoryUsage(info: MemoryUsage) {
+        let numBytes = 0;
+
+        for (const block of this.unicodeBlocks) {
+            numBytes += (block.max - block.min) * 2;
+        }
+
+        let textureBytes =
+            this.m_glyphTextureCache.textureSize.x * this.m_glyphTextureCache.textureSize.y * 3;
+
+        for (const page in this.m_loadedPages.entries) {
+            if (this.m_loadedPages.get(page) !== undefined) {
+                const loadedPage = this.m_loadedPages.get(page);
+                if (loadedPage !== undefined) {
+                    textureBytes += loadedPage.image.width * loadedPage.image.height * 3;
```

Comment:
```suggestion
                    textureBytes += loadedPage.image.width * loadedPage.image.height * 4;
```

Loaded textures are ``.png``, and thus probably contain an alpha channel.

Reply:
They are actually RGB, whiy would they need an alpha channel? The PNG format does not require it...

Reply:
Fixed